app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -59,7 +59,7 @@ def proofer
 
       def add_proofing_component
         ProofingComponent.
-          create_or_find_by(user: effective_user).
+          create_or_find_by(user: current_or_hybrid_user).
```

Comment:
This should probably just be `current_user`, right? Can someone from @18F/identity-joy confirm?